<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="sidebar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2596be</string>
    </property>
    <widget class="QWidget" name="wid1" native="true">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>59</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;
</string>
      </property>
      <property name="text">
       <string>Neu Termin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wid1_2" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Remove Termin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wid1_3" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);</string>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Update Termin</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>531</width>
      <height>481</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Datum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Uhr</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gift</string>
     </property>
    </column>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>51</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
